src/main/java/com/bp3/camunda/camunda7/C7RESTConnector.java

```diff
@@ -104,7 +104,11 @@ public void execute(ExternalTask externalTask, ExternalTaskService externalTaskS
 
             // complete the external task
             externalTaskService.complete(externalTask, variables);
-        } catch(ConnectorException e) {
+        }
+        // All exceptions need to be caught, so we can handle them gracefully, otherwise they get swallowed
+        // or the task worker will keep getting the same request, and it might just keep rolling around with
+        // the same exception
+        catch(Throwable e) {
```

Comment:
This should probably be Exception rather than Throwable. As you don't want to catch Errors like out of memory